### DISCUSSION TITLE
Server discord Nuked - Need Assistance

### DISCUSSION THREAD

**Discussion by Mgattina** [community/community#129676] [Other Feature Feedback, Questions, & Ideas]
Hello,

Recently, my Discord server was maliciously nuked by unknown individuals. This has caused significant disruption and frustration. At the moment, I only have their Discord account information.

I am reaching out to see if it's possible to gather any additional information that could help identify these individuals. If there are any details or steps you can provide to assist in stopping these actions, I would greatly appreciate it.

Thank you for your attention to this matter.
![Screenshot 2024-06-25 003815](https://github.com/community/community/assets/124304134/bd246328-9e0f-4efb-9e26-acdfc8cabc1c)

[image removed by moderator]

**Comment by LiteBrite82** [comment]
Hi @Mgattina ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
